app/controllers/responsiblepeople/FitAndProperController.scala

```diff
@@ -61,7 +63,7 @@ class FitAndProperController @Inject()(
           case ValidForm(_, data) => {
             for {
               _ <- updateDataStrict[ResponsiblePerson](index) { rp =>
-                rp.hasAlreadyPassedFitAndProper(Some(data))
+                rp.approvalFlags(ApprovalFlags(hasAlreadyPassedFitAndProper = Some(data)))
```

Comment:
This needs to be copied, otherwise you will lose the existing approvalcheck information